bin/mvmath.py

```diff
@@ -69,7 +69,7 @@ def validate_args(keywords, argvals):
     MANDATORY_OPTIONS = []
 
     if len([x for x in argvals if x in MANDATORY_OPTIONS]) < len(MANDATORY_OPTIONS):
-        die("Insuffucient number of mandatory arguments found. Mandatory argumets are: %s" % MANDATORY_OPTIONS)
+        die("Insuffucient number of mandatory arguments found. Mandatory argumets are: %s" % (MANDATORY_OPTIONS,))
```

Comment:
is that extra coma at the end a typo?

Reply:
No, the syntax is valid.    It means tuple with a single value.  Similarly to how, `x = "value",` makes a tuple containing a single string, which is identical to `x = ("value",)`, but in the modulus context  I think the `()`s are required.... I'm sure it ties back to order of operations.)

Frankly, this is probably me overthinking it.  I've been bitten by subtle bugs like this in the past so I'm probably overly sensitive to the scenario.  I just reviewed the [docs](https://docs.python.org/2/library/stdtypes.html#id16), it appears that this only matters for tuples, so because `MANDATORY_OPTIONS` is a list it doesn't matter.  (But if MANDITORY_OPTIONS type was changed to tuple, then the formatting would fail, unless it contained exactly 1 item.)

So this change isn't necessary; but I suppose it's all academic since this line will never be called, without mandatory options defined (so really lines 69-72 aren't really need at all)

Reply:
I'd be glad to update the PR branch, if you'd prefer that I revert.

Reply:
i think the changes are fine. let me spend bit of time testing this && running through App Inspect again, at which point of time we can merge this. more updates shortly.